### ISSUE TITLE
2019 Winter Event Support

### ISSUE THREAD

**Issue by mrmin123** [perryhuynh/kcauto#331]
~~Soon(tm)~~

Untested and WIP branch is here: [`331-2019-winter-event`](https://github.com/mrmin123/kcauto/tree/331-2019-winter-event)

Via command line:
```
git fetch
git checkout 331-2019-winter-event
```
or, if you have multiple remotes,
```
git fetch
git checkout -b 331-2019-winter-event origin/331-2019-winter-event
```

**Comment by mrmin123** [comment]
They've removed the node select dialogue popup on the map. Changes in this branch will address this.

**Comment by mrmin123** [comment]
Event support has been released in [7.2.0](https://github.com/mrmin123/kcauto/releases/tag/7.2.0). Further tweaks, if necessary, will continue to happen in the [`331-2019-winter-event`](https://github.com/mrmin123/kcauto/tree/331-2019-winter-event) branch.

**Comment by mrmin123** [comment]
Two more things that need to be addressed:

* webUI does not support node ZZ (ZZ1 is the closest thing)
* There are reports that E-2 coords for node P is a bit off, causing LBAS node assignment issues from time to time; try coords `1030, 438`
